streaming/src/main/scala/io/gearpump/streaming/task/ExpressTransport.scala

```diff
@@ -94,7 +94,9 @@ class SendLater(express: Express, serializerPool: SerializerPool, sender: ActorR
       flushPendingMessages(localActor.get, transportId)
     } else {
       val remoteAddress = express.lookupRemoteAddress(transportId)
-      flushPendingMessages(remoteAddress.get, transportId)
+      if (remoteAddress.isDefined) {
```

Comment:
this is dropping messages which is bad. I'd better fix the empty remoteAddress issue.